From 29174bb397f1c1821f93ef5dbdfec159bfbd550c Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Sat, 10 Dec 2016 14:29:52 +0000
Subject: [PATCH 119/126] e1000e: change default policy

change the default irq mitigation policy for e1000e to be
more HPC/cluster friendly
---
 drivers/net/ethernet/intel/e1000e/param.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/intel/e1000e/param.c b/drivers/net/ethernet/intel/e1000e/param.c
index 6d8c39abee16..ef1122ad3b98 100644
--- a/drivers/net/ethernet/intel/e1000e/param.c
+++ b/drivers/net/ethernet/intel/e1000e/param.c
@@ -92,7 +92,7 @@ E1000_PARAM(RxAbsIntDelay, "Receive Absolute Interrupt Delay");
  * Valid Range: 100-100000 or one of: 0=off, 1=dynamic, 3=dynamic conservative
  */
 E1000_PARAM(InterruptThrottleRate, "Interrupt Throttling Rate");
-#define DEFAULT_ITR 3
+#define DEFAULT_ITR 1
 #define MAX_ITR 100000
 #define MIN_ITR 100
 
-- 
2.13.2

